lnwallet/channel.go

```diff
@@ -5578,14 +5578,24 @@ func (lc *LightningChannel) ForceClose() (*LocalForceCloseSummary, error) {
 	lc.Lock()
 	defer lc.Unlock()
 
+	// If the channel isn't in the default channel state, then we won't
+	// allow a force close, as it may be the case that we have a dated
+	// version of the commitment, or this is actually a channel shell.
+	if lc.channelState.HasChanStatus(channeldb.ChanStatusLocalDataLoss) {
```

Comment:
Comment and returned error says non-default state, while this checks for `DataLoss` only. What's correct?

Reply:
Data loss, comment was left over from a check that was too broad which ended up introducing a test failure.